DigitalLearningSolutions.Data/Services/DelegateDownloadFileService.cs

```diff
@@ -59,10 +110,31 @@ private void PopulateDelegatesSheet(IXLWorkbook workbook, int centreId)
                     x.Answer5,
                     x.Answer6,
                     x.Active,
-                    x.EmailAddress
+                    x.EmailAddress,
                 }
             );
 
+            if (!delegateRecords.Any())
+            {
+                var emptyDelegate = new

```

Comment:
As this file is used in the upload journey, I was unsure if I should leave things blank, or include suitable details for a fake delegate to serve as an example for the user. No idea how likely this will be as a scenario, but for now, all fields will be blank except the Job Group ID and Active columns.

Reply:
I suppose that makes sense though it feels a little weird to me to pre-populate only two values, and to default Job Group ID to 1. I think making a fake delegate with all details filled in would risk the user forgetting to delete the fake delegate and registering them in the upload process.

Reply:
I think we need to default to ID 10 - Other for the job group if we're going to default to anything

Reply:
A good idea, but there should always be a delegate at any centre with an admin (because they should have a delegate account), so I don't think this will ever be used.

I don't think we'll ever have the case where the centre only has admins and those admins don't have candidate accounts - only admins that registered before November will be in this state, and they'll have delegates by now if they're using the system at all.

Reply:
Seeing as it is incredibly unlikely that this file will be empty, I've removed the fall-back code. Additionally, admins can add delegates the old fashioned way from the "Register" button, which will allow the download to work fine. Though this isn't really obvious to a user.

@SteveJacksonSoft I presume your comment above about how this shouldn't be a problem takes into account your slack message from earlier saying some admins might not have delegate accounts yet?

Reply:
Ah yeah, you can imagine the slack comment in between the two paragraphs of my comment 😂 
Basically, yes some won't have delegate accounts, but those centres will almost definitely have delegates at them.